index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <!-- meta data -->
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+
+    <!-- page title -->
+    <title>News Blog</title>
+
+    <!-- custom stylesheet -->
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="d-flex col">
+    <header class="d-flex col center">
+        <div class="d-flex col center">
+            <span id="title">News Blog</span>
+            <span id="sub-title">worldwide news</span>
+        </div>
+    </header>
+
+    <main>
+        <section id="articles-header" class="d-flex row">
+            <div class="d-flex col" id="category">
+                <label for="filter">select category</label>
+                <select id="filter">
```

Comment:
It would be good if we can give relative id not generic. Let's say if you have another filter on the page then what you will do.

id = "categoryFilter" would be the best fit as an Id.

Reply:
ok